packages/react/src/components/Reaction/Reaction.tsx

```diff
@@ -29,17 +29,17 @@ export interface ReactionSlotClassNames {
 export interface ReactionProps
   extends UIComponentProps<ReactionProps>,
     ChildrenComponentProps,
-    ContentComponentProps<ShorthandValue> {
+    ContentComponentProps<ShorthandValue<BoxProps>> {
   /**
    * Accessibility behavior if overridden by the user.
    */
   accessibility?: Accessibility
 
   /** A reaction can have icon for the indicator of the reaction. */
-  icon?: ShorthandValue
+  icon?: ShorthandValue<IconProps>
 
   /** A reaction can have content shown next to the icon. */
-  content?: ShorthandValue
+  content?: ShorthandValue<BoxProps>
```

Comment:
I think we can actually remove this, as we already have it defined when we are extending `ContentComponentProps<ShorthandValue<BoxProps>>`